core-war/src/main/webapp/admin/jsp/newRegistration.jsp

```diff
@@ -216,8 +210,18 @@
 
       <view:directoryExtraForm userId="unknown" edition="true"/>
 
-      <div id="QapTcha"></div>
-      <a href="#" class="btn-registrer submit" onclick="saveNewUser()">
+      
+      
+      <div class="slidercaptcha card">
+  	
+        <div class="card-body">
+    		<div id="captcha"></div>
+  	</div>
+      </div>
+
+      
+      
+      <a href="#" id="btn-register" class="btn-registrer submit" onclick="">
```

Comment:
According to the Sonar feedback, and in order to satisfy the accessibility recommendations, the <a> tag here should be replaced by a <button> one (with the CSS class link  if the button has to appear like an anchor link)